Update side gig policy (#8652)
PostHog/posthog.com: contents/handbook/people/side-gigs.md

In a few cases, you may want your side gig to become your full time work one day. That is ok - please just let us know, so we can create a plan. We know the key to motivated people is to help you achieve your long term goals, and to align this with what PostHog needs, whether or not you eventually achieve them with us.

Above everything else, if you are going above and beyond for PostHog and you're still able to look after yourself properly, side gigs (whether paid or unpaid) are totally fine. We don't think that's possible beyond a certain level of time/energy commitment to them, but we are very happy for you to spend a little time on them each week.

## Intellectual property

Just to reassure you, PostHog won't try to claim ownership of any intellectual property (IP) you create in your personal time, e.g. if you are contributing to another open source project as a hobby. However, you need to be _really_ careful that you do not introduce any of PostHog's non-open source IP into any project that you work on - this can cause serious legal headaches. As a rule, anything from PostHog that is explicitly MIT-licensed is fine to use, anything else is not.

> If you are ever worried about this, please talk to Fraser and he can help you figure out the best solution here. 

Author: ```suggestion
If you are ever worried about this, please talk to Fraser and he can help you figure out the best solution here, especially if what you are working on directly competes with something PostHog has built or is on our roadmap. 
```

showTitle: true
---

PostHog looks for passion in the people it hires. This often correlates with people who have side projects as a hobby. For example, we view pre-existing open source work as a strong qualifier that you're good enough at programming that it's fun to do rather than frustrating and hard!

These side gigs may sometimes earn you money. Sometimes, you may one day want your side gig to become your main gig.

We have deliberately called them "side gigs", as we are ok with you earning money on the side. We are not ok with this being your main focus and PostHog being just a paycheck. Quite simply, we are too small for PostHog not to be your main motivation. For this reason, we also currently don't offer part time work as an option at PostHog. 

## Managing time

The key distinction to something being a side gig, and thus it being appropriate, is its impact on your work and the amount of time involved.

> A few hours a month on a paid side gig is acceptable. Over 10 hours a week on a paid or open source side gig is a significant amount of extra work which we would expect to start impacting your performance. In any case, side gigs should by default be something you work on in your personal time. 

Reviewer: ```suggestion
A few hours a month on a paid side gig is acceptable. In any case, side gigs should by default be something you work on in your personal time, and they should not impact the work you do at PostHog
```

> If you are doing paid speaking, the exception based on time does not apply so long as you make it clear that you work for us - we view this as a great way to get PostHog's name out there.

Reviewer: ```suggestion
```

In a few cases, you may want your side gig to become your full time work one day. That is ok - please just let us know, so we can create a plan. We know the key to motivated people is to help you achieve your long term goals, and to align this with what PostHog needs, whether or not you eventually achieve them with us.

Above everything else, if you are going above and beyond for PostHog and you're still able to look after yourself properly, side gigs (whether paid or unpaid) are totally fine. We don't think that's possible beyond a certain level of time/energy commitment to them, but we are very happy for you to spend a little time on them each week.

> ## Intellectual property

Reviewer: Is there something around not directly competing with something PostHog has built or might build in the future? Maybe a "chat with us if you think your side project might fall under this"?

Author: Added something on this
